Classes/Controller/ModuleController.php

```diff
@@ -442,6 +442,10 @@ public function ajaxUsersImport(ServerRequestInterface $request, Response $respo
 
         if ($success) {
             list($filter, $baseDn) = Authentication::getRelativeDistinguishedNames($params['dn'], 2);
+
+            // Escape Backslashes for LDAP-Search
+            $filter = str_replace('\\','',$filter);
```

Comment:
this is not sufficient, see https://github.com/xperseguers/t3ext-ig_ldap_sso_auth/issues/64#issuecomment-547802724